apps/open-swe/src/graphs/planner/nodes/take-action.ts

```
@@ -76,130 +76,78 @@ export async function takeActions(
+    lastMessage.tool_calls.map(async (toolCall) => {
+      const tool = allTools.find((t) => t.name === toolCall.name);
+      if (!tool) {
+        return new ToolMessage({
+          tool_call_id: toolCall.id,
+          content: `Tool ${toolCall.name} not found.`,
+        });
       }
+      try {
+        const toolOutput = await tool.invoke(toolCall.args, {
+          ...config,
+          runId: uuidv4(),
```

Biome (2.3.14)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)